apps/fishing-map/features/search/Search.tsx

```diff
@@ -40,82 +23,59 @@ import {
   setVesselGroupCurrentDataviewIds,
 } from 'features/vessel-groups/vessel-groups.slice'
 import { TrackCategory, trackEvent } from 'features/app/analytics.hooks'
+import SearchBasic from 'features/search/SearchBasic'
+import SearchAdvanced from 'features/search/SearchAdvanced'
+import SearchPlaceholder from 'features/search/SearchPlaceholders'
+import { WORKSPACE } from 'routes/routes'
+import I18nNumber from 'features/i18n/i18nNumber'
 import {
   fetchVesselSearchThunk,
-  selectSearchResults,
   cleanVesselSearchResults,
-  selectSearchStatus,
-  selectSearchStatusCode,
   VesselWithDatasets,
   RESULTS_PER_PAGE,
-  checkSearchFiltersEnabled,
   resetFilters,
   setSuggestionClicked,
-  SearchType,
   SearchFilter,
+  selectSearchOption,
+  selectSearchPagination,
+  selectSearchResults,
 } from './search.slice'
-import styles from './Search.module.css'
-import SearchFilters from './SearchFilters'
 import { useSearchConnect, useSearchFiltersConnect } from './search.hook'
-import SearchPlaceholder, {
-  SearchNotAllowed,
-  SearchNoResultsState,
-  SearchEmptyState,
-} from './SearchPlaceholders'
 import {
-  isBasicSearchAllowed,
-  isAdvancedSearchAllowed,
   selectBasicSearchDatasets,
   selectAdvancedSearchDatasets,
+  isBasicSearchAllowed,
+  isAdvancedSearchAllowed,
 } from './search.selectors'
+import styles from './Search.module.css'
 
 const MIN_SEARCH_CHARACTERS = 3
 
 function Search() {
-  const { t, i18n } = useTranslation()
+  const { t } = useTranslation()
   const dispatch = useAppDispatch()
   const urlQuery = useSelector(selectSearchQuery)
   const { addNewDataviewInstances } = useDataviewInstancesConnect()
+  const basicSearchAllowed = useSelector(isBasicSearchAllowed)
+  const advancedSearchAllowed = useSelector(isAdvancedSearchAllowed)
   const [searchQuery, setSearchQuery] = useState((urlQuery || '') as string)
+  const searchResults = useSelector(selectSearchResults)
   const { searchFilters } = useSearchFiltersConnect()
-  const { searchPagination, searchSuggestion, searchSuggestionClicked } = useSearchConnect()
+  const { searchPagination, searchSuggestion } = useSearchConnect()
   const debouncedQuery = useDebounce(searchQuery, 600)
-  const { dispatchQueryParams } = useLocationConnect()
+  const { dispatchQueryParams, dispatchLocation } = useLocationConnect()
   const heatmapDataviews = useSelector(selectActiveHeatmapDataviews)
-  const basicSearchAllowed = useSelector(isBasicSearchAllowed)
-  const advancedSearchAllowed = useSelector(isAdvancedSearchAllowed)
-  const searchResults = useSelector(selectSearchResults)
-  const searchStatus = useSelector(selectSearchStatus)
-  const searchStatusCode = useSelector(selectSearchStatusCode)
   const gfwUser = useSelector(isGFWUser)
-  const hasSearchFilters = checkSearchFiltersEnabled(searchFilters)
-  const vesselDataviews = useSelector(selectVesselsDataviews)
   const [vesselsSelected, setVesselsSelected] = useState<VesselWithDatasets[]>([])
```

Comment:
As we need to drill down this state through Search => SearchResults, do you think worth it to move to a store reducer?

Reply:
makes sense